lasy/utils/mode_decomposition.py

```diff
@@ -9,7 +9,15 @@
 from lasy.utils.exp_data_utils import find_d4sigma
 
 
-def hermite_gauss_decomposition(laserProfile, wavelength, m_max=12, n_max=12, res=1e-6):
+def hermite_gauss_decomposition(
+    laserProfile,
+    wavelength,
+    m_max,
+    n_max,
```

Comment:
I think it's good to keep the default 12 for these two, actually, people may wonder what makes sense. That indeed requires one change in the order of parameters in the doc.

Reply:
Requested changes have been made:)